README.md

```diff
@@ -115,6 +115,78 @@ An XML variant may look like:
 </int-aws:s3-inbound-channel-adapter>
 ````
 
+###Streaming Inbound Channel Adapter
+
+This adapter produces message with payloads of type `InputStream`, allowing S3 objects to be fetched without writing to the local file system. Since the session remains open, the consuming application is responsible for closing the session when the file has been consumed. 
```

Comment:
It is better to keep each sentence on its own line: http://asciidoctor.org/docs/asciidoc-recommended-practices/#one-sentence-per-line